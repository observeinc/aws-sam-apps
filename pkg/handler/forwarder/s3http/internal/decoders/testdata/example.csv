a,b,c
1,2,
hello,world,nope
"24""",test\help
