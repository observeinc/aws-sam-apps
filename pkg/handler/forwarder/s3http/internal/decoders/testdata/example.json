{
    "hello": "world",
    "anum": 3
}
