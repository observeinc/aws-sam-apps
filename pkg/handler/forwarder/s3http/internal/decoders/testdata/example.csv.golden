{"a":"1","b":"2"}
{"a":"hello","b":"world","c":"nope"}
{"a":"24\"","b":"test\\help"}
